### ISSUE TITLE
organizational communication, group memory, and welcome rituals

### ISSUE THREAD

**Issue by dckc** [rchain-community/sm19a#4]
I'm pleased at the increased quality of discussion in the **#members** channel recently. That and chatting with @richjensen3000 reminded me of some patterns I'd like to emphasize:

  - channels go well with periodic rituals and guides
    - Greg's weekly debrief is a well-established periodic ritual, but the connection to any particular discord channel is weak. I'd like one or more guides to, for example, collect items from the channel over the course of the week as input to the debrief and to take some notes in the channel during the debrief, especially to track and ensure follow-up and make updates to group memory

more later on welcome rituals, async communications (discourse forum? loomio?), and group memory (mediawiki?)

Meanwhile, see, for example, the way the indieweb community uses IRC + mediawiki: https://indieweb.org/IRC

Also see: [RChain Collaboration](https://docs.google.com/presentation/d/1KCO48W6e2voHUapr7ZB0xASnPcogeFxnNrGbFcJwHK4/edit#slide=id.p) slides

**Comment by faddat** [comment]
A discourse forum would be a very, very healthy development.

**Comment by dckc** [comment]
@jimscarver says "count me in"

**Comment by dckc** [comment]
At about 40 minutes into the May 14 session, @leithaus said while he's happy to go through the process of accountability and diligence, he's frustrated at being asked to to go over things that have been gone over many times. And yet, where is the process of accountability and diligence documented? How should we expect new members to find it? If I start at https://rchain.coop/ , is there a straightforward path through a reasonable number of links to get me to it? The main communication channel of the coop is Greg's weekly debrief. When I first started looking into RChain, it looked like I was expected to watch 40+ hours of video to come up to speed. Fortunately, in an obscure corner of the organization, I found a newsletter archive including an index of topics. I organized these into [a wiki index](https://github.com/rchain/bounties/wiki/Weekly-Debrief-Index) ; fortunately, this has largely been subsumed by the [community-debriefs blog section](https://blog.rchain.coop/category/community/community-debriefs/) . Unfortunately, our blog is unusably slow: it's [rated F for performance](https://github.com/rchain/marketing/issues/54), as I pointed out in October.

In early 2018, many of us spent a few hundred hours or so in total organizing a web site for members to efficiently find what they need and collaborate (e.g. [4 Week UI/UX Mock Ups](https://github.com/rchain/bounties/issues/446) ). But when we asked to actually take it live, the decision process by the coop staff took months, with very little setting of expectations nor clarity about who had the ball. Ultimately, the answer was no... or at least: no answer for sufficiently long that everyone lost motivation.

In the Oct 2018 annual member meeting, @leithaus presented a slide with [rnode releases plotted on top of a RHOC price chart][s1] and asked (somewhat rhetorically), why doesn't the market respond to the excellent progress of the core dev team? Well, let's see... do we feature those releases on the coop homepage? No. How long does it take to find them in our F-performance blog? A google search for "rchain releases" does turn up https://github.com/rchain/rchain/releases , but there's precious little copy there to tell anyone why the software is interesting.

I worked with @pmoorman on a [library of RChain](https://github.com/rchain/bounties/issues/1004) to feature this information (one draft: https://rchain-info.netlify.com/ ). We weren't able to convince @derekberes and @ned-robinson that an efficient site with technically correct information was more important than videos and animations. In some sense it's just a difference of style and opinion, but my 25+ years of collaboration and management using the web tells me it has a real day-to-day impact on efficiency of collaboration in the membership and the community.

[s1]: https://youtu.be/xdi6ZCsX4K8?t=6266

**Comment by dckc** [comment]
@jimscarver writes "The trust metric is designed to delegate power from the top down. This compliments bottom up liquid democracy. " in [Liquid Democracy+ Prototype in RhoLang](https://docs.google.com/document/d/14JJPWtrFxkxzHa1NAmwzhTmeQO1vJtKDOJgBB_MPyfY/edit#heading=h.ygdnkypn002t)

That strikes a chord w.r.t. this issue. I wonder if the sm19a 100+ member "phone tree" has gone cold or if it's worth investing in.

**Comment by jimscarver** [comment]
Yes! I am hoping that our governance tools like liquid democracy and trust metrics might become part of our culture and help define our connections revealing the effective structure of our organization.

We might discuss the phone tree idea at office hours. The normal purpose of a phone tree traditionally was to broadcast important messages.  It feels like this personal communication does a lot more than a @all notification but it's not clear how exactly this connects to this issue and trustworthy governance.
1. peer to peer communication is the basis for coordinated action and collective being. Thoughts:
2. arbitrary connections seed peer to peer relations
3. the largest success factor in open source projects has been shown to be strong personal relations.
3. communities or circles of trust are a natural decentralization of the larger community.
4. trust, top down or bottom up is tested by action, Trust is hard to build but easy to lose. Communicating intention before action in trust circles may lead to better action and increased trust.

**Comment by dckc** [comment]
bookmarking: https://rholang.github.io/

p.s. March 11: I submitted a few pull requests: https://github.com/rholang/rholang.github.io/pulls

Let's see how the collaboration goes...

**Comment by dckc** [comment]
> A discourse forum would be a very, very healthy development.

Yes, but we're extremely thin on resources just now.

I lean toward stackoverflow for now. It's really, really well organized. I just got yet another obscure question answered better than I would have expected:

> I want sqldeveloper to *ding!* when a long query is done; anybody know how to do this? 
>
> never fear! stackoverflow is here! https://stackoverflow.com/a/54968096/7963 Alex Poole Mar 3 '19

----

as I edit some [FAQs](https://github.com/rholang/rholang.github.io/labels/FAQ) I wonder about pushing them to stackoverflow / stackexchange. I find **cryptocurrency** tags on two stackexchange sites:

 - [Newest 'cryptocurrency' Questions \- Cryptography Stack Exchange](https://crypto.stackexchange.com/questions/tagged/cryptocurrency)
 - [Newest 'cryptocurrency' Questions \- Personal Finance & Money Stack Exchange](https://money.stackexchange.com/questions/tagged/cryptocurrency)

plus a meta discussion...

 - [Stack Exchange website for cryptocurrencies in general \- Meta Stack Exchange](https://meta.stackexchange.com/questions/271576/stack-exchange-website-for-cryptocurrencies-in-general)

the meta discussion concludes by recommending [Bitcoin Stack Exchange](https://bitcoin.stackexchange.com/)

**Comment by dckc** [comment]
@jimscarver did you say Darryl is heading a communications group? Does it have a home in the web?

p.s. Jim notes [Communications Folder](https://drive.google.com/drive/folders/1kGsXnB1yx_4oJZIWITymE29WvbwKsuf8)

**Comment by SteveHenley** [comment]
From my understanding the communication working group that Darryl leads is more about coop communications. 

I will add stack overflow to the Tech Governance agenda. 


- Steve Henley

- Sent via my iPhone

**Comment by dckc** [comment]
> From my understanding the communication working group that Darryl leads is more about coop communications.

This issue is about coop communications too.

**Comment by dckc** [comment]
> ... I will add stack overflow to the Tech Governance agenda.

I'm not sure I've told anybody in that group enough to represent my position. Please stand by until I have.

Meanwhile, again, does that group have a home in the web? For example, something that lists its membership?

**Comment by dckc** [comment]
- [Microcontent: How to Write Headlines, Page Titles, and Subject Lines](https://www.nngroup.com/articles/microcontent-how-to-write-headlines-page-titles-and-subject-lines/)
 - [Prioritize: Good Content Bubbles to the Top](https://www.nngroup.com/articles/prioritize-good-content-bubbles-to-the-top/)

@SteveHenley  agreed to relay micrcontent / bubbling advice to RChain release notes writers

for example: [0.9.23 release notes](https://github.com/rchain/rchain/releases/tag/v0.9.23) should highlight the fact that nodes can catch up because [PR 2902](https://github.com/rchain/rchain/pull/2902 ) is included (if that's actually the case; https://github.com/rchain-community/rstake/issues/4 is still open because we're not sure.)

**Comment by dckc** [comment]
anybody know who runs https://rchain-community.readme.io/ ? is it backed by a public git repo?

**Comment by nzpr** [comment]
@dckc#3040 https://rchain-community.readme.io/ I created that one, its not powered by git repo, anyone can suggest edits via their own interface.

**Comment by dckc** [comment]
Several of us had a very promising meeting yesterday;
 - [RChain Communications Working Group Log](https://docs.google.com/document/d/1yrCqj5GWKElRVGYrPcrRDOEAZl4ILk3RYv00GFo6Yts/edit#)

I took the ball to review / revise a draft charter:

 - [RChain Co\-op Communications Committee Charter \- v2 \- Google Docs](https://docs.google.com/document/d/1i-W9MUG_W5y3mVQ8NjwJ-1EklCArzisGdQjpAzwLc2Q/edit?pli=1)

The coop adopting that charter would resolve this issue to my satisfaction.
